packages/@okta/vuepress-site/docs/release-notes/2019-10-2/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Okta API Products Release Notes
+---
+
+## 2019.10.2
+
+| Change                                                              | Expected in Preview Orgs |
+|---------------------------------------------------------------------|--------------------------|
+| [User Types Error Message Change](#user-types-error-message-change) | October 30, 2019         |
+| [Bugs Fixed in 2019.10.2](#bugs-fixed-in-2019-10-2)                 | October 30, 2019         |
+
+### User Types Error Message Change
+
+Error messages returned by the `/types/user` [API](/docs/reference/api/user-types/) have changed. Omitting display name or variable name when attempting to create a User Type, or specifying a variable name that is already in use, results in a more specific error message being returned. <!-- OKTA-241017 -->
```

Comment:
I think we should refer to this as the User Types API (like the title of the page). 

Same for the Features API below.

Reply:
Made the change